src/solver/solve.jl

```diff
@@ -74,6 +75,15 @@ function solve_atmos!(simulation)
         return AtmosSolveResults(nothing, :simulation_crashed, nothing)
     finally
         # Close all the files opened by the writers
+
+        maxrss_bytes = maxrss()
+        maxrss_str =
+            maxrss_bytes > 2^30 ?
+            "$(round(maxrss_bytes / 2^30, digits = 2)) GiB" :
+            "$(round(maxrss_bytes / 2^20, digits = 2)) MiB"
```

Comment:
```suggestion
        maxrss_str = prettymemory(maxrss())
```
Maybe we can just reuse `prettymemory` here?

Reply:
Oh nice! :)